CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 - Generated OpenAPI now describes whether each endpoint is forwarded (#3935).
+- The "Node Output" page has been relabelled as "Troubleshooting" in the documentation and CLI commands for troubleshooting have been added to it.
```

Comment:
We don't normally write documentation changes to the changelog because it's already very verbose. However, we should find a way to flag these somehow. If you have any suggestions, they are welcome!

Reply:
How does adding a ​`doc/DOC-CHANGELOG.md`​ where docs changes can be tracked sound? 

I could be convinced that we have versioned docs already and so the old versions of docs act as a history. However, I do think there's not much overhead to maintaining a docs changelog and it will be useful to users who are already familiar with a version of CCF. It will towards those users towards updated or new areas of documentation that they should pay attention to when interaction with a different version of CCF.

Reply:
I've taken the `### Documentation` suggestion